api/config/config_test.json

```diff
@@ -172,13 +172,20 @@
   "e2eTestUsers": {
     "workspaceDeletionBatchSize": 15,
     "testUserEmails": [
+      "puppeteer-tester-1@fake-research-aou.org",
```

Comment:
This list is for the deletion scripts.  I thought it would be good to clean up older users too.

Reply:
How many users did you add? It looks like the old version of the file did not include puppeteer-tester-1, puppeteer-tester-2, or puppeteer-tester-4 through puppeteer-tester-6.

Reply:
for each user whatever-N, I added 1 to N here